### DISCUSSION TITLE
page hosting inside poxy manager

### DISCUSSION THREAD

**Discussion by Kowy54** [NginxProxyManager/nginx-proxy-manager#1327] [Q&A]
Can i put custom webpage to proxy_manager directory and redirect from domain to that page?

**Comment by jc21** [comment]
Technically this is possible, you can mount some folder to the npm container and then write some advanced nginx config inside the proxy host. You have to specify a location / to override any generated config

```
root /path/to/www;

location / {
  index index.html;
  try_files $uri /index.html;
}
```

I haven't tested this though

**Comment by chaptergy** [comment]
Just wanted to mention [this feature request](https://github.com/jc21/nginx-proxy-manager/issues/58) to be able to host files directly, since it already contains useful information in this topic.
